### PR TITLE
Improve error message on lab1 feed back

### LINKED ISSUE #45: Improve error message on lab1 feed back.
Currently error message if feedback collection is not active looks like: `:feedback-collecting-disabled "Сбор отзывов отключен (слишком рано или слишком поздно)."` which is correct, but not too informative.

We should show something like:

```clojure
Сбор отзывов отключен. Ваша группа докладов: _____________. Сейчас: ________________. Ожидаемые сроки отзывов:

- ___________ -- ____________
- ___________ -- ____________
- ___________ -- ____________
- ___________ -- ____________
```

We need to list intervals for all lessons of the group.

### PR DESCRIPTION
**Pull request by Ivanio1**
Close #45

### CODE REVIEW

**Comment by Ivanio1** [comment]
@ryukzak

File: test/course_bot/presentation_test.clj [hunk 1]
```diff
@@ -631,7 +631,7 @@
 
       (with-redefs [misc/today (fn [] (misc/read-time "2022.01.01 11:29 +0000"))]
         (is (answers? (talk 1 "/lab1feedback")
-                      "Lesson feedback is not available.")))
+                      "Lesson feedback is not available. Your lab1 group: lgr1. Now: 2022.01.01 11:29 +0000. Expected feedback dates:\n- 2022.01.01 12:30 +0000 -- 2022.01.01 15:00 +0000\n- 2022.01.02 12:30 +0000 -- 2022.01.02 15:00 +0000")))
```
Comment (ryukzak) [inline test/course_bot/presentation_test.clj:634]: Use `tt/unlines` instead of `\n`

File: src/course_bot/presentation.clj [hunk 2]
```diff
@@ -591,7 +591,10 @@
                                         (->> pass-lessons
                                              (map #(str "- " (:datetime %)))
                                              (str/join "\n"))))
-                (talk/send-text token id (tr :pres/lesson-feedback-not-available))))
+                (let [lessons (->> (lessons pres-conf pres-group))]
+                  (talk/send-text token id (format (tr :pres/lesson-feedback-not-available-:pres-group-:now-:right-times-list) pres-group (misc/str-time now) (->> lessons
+                                                                                                                                                                   (map #(str "- " (misc/str-time (+ (* 30 60 1000) (misc/read-time (:datetime %)))) " -- " (misc/str-time (+ (* 180 60 1000) (misc/read-time (:datetime %))))))
+                                                                                                                                                                   (str/join "\n")))))))
```
Comment (ryukzak) [inline src/course_bot/presentation.clj:597]: Reduce line length and make it readable

**Review by ryukzak — CHANGES_REQUESTED** [review]

**Comment by Ivanio1** [comment]
@ryukzak All fixed

**Comment by ryukzak** [comment]
Small fix for your code: https://github.com/ryukzak/course-bot/commit/ac1b9a864974800df2675c9d083d1250819a1e4e

**Pull request merged**